test/games/patchers/test_csharp_subprocess.py

```diff
@@ -35,8 +42,13 @@ def test_is_windows(mocker, system_name, expected):
 
 
 @pytest.mark.parametrize("add_mono", [False, True])
-def test_process_command_no_thread(mock_is_windows, mocker: pytest_mock.MockerFixture, monkeypatch,
+@pytest.mark.skipif(mock_is_windows and mock_is_mac,
```

Comment:
Does this actually work?

Reply:
Double checked, it doesn't. Whoops.
That currently causes it to skip the test in its entirety.

Reply:
just do a `if a and b: pytest.skip()` in the function body

Reply:
Addressed now
![grafik](https://github.com/randovania/randovania/assets/38186597/c55acc71-677a-4bf6-8c50-0d2f1f06e8fc)